AdyenActions/Components/3DS2/Action handlers/AnyThreeDS2ActionHandler.swift

```diff
@@ -48,7 +50,7 @@ internal func createDefaultThreeDS2CoreActionHandler(
     delegatedAuthenticationConfiguration: ThreeDS2Component.Configuration.DelegatedAuthentication?
 ) -> AnyThreeDS2CoreActionHandler {
     #if canImport(AdyenAuthentication)
-        if #available(iOS 14.0, *), let delegatedAuthenticationConfiguration {
+        if #available(iOS 16.0, *), let delegatedAuthenticationConfiguration {
             return ThreeDS2PlusDACoreActionHandler(context: context,
```

Comment:
So previous `??` check was not needed then? Seemed like there was some checking and returning a different one etc

Reply:
It wasn't needed indeed. It was assuming that initializer for the AuthenticationService(Authentication framework) was throwing, but in the latest release it is not throwing. So there is no fall back required at this point.